core/src/main/java/com/xmartlabs/base/core/module/PicassoModule.java

```diff
@@ -17,9 +17,11 @@
 
 @Module
 public class PicassoModule {
```

Comment:
Well, it depends on whether the "service" module should just hold retrofit stuff or the whole layer. If we change that, I'll move it